doc/development-workflow.md

```diff
@@ -0,0 +1,22 @@
+# Development workflow
+
+## PR author
+
+1. Open a PR with the changes you want to merge.
+
+2. Add the "Waiting on factcheck" label if the changes require feedback from someone other than developers.
+
+3. Move the associated Trello card from "In progress" to "Code review".
```

Comment:
> I think @chrisroos already knows about the change in order of the workflow, but is unsure if/when we should add the "Waiting on factcheck label". Is that right, Chris?

That is correct, @floehopper / @leenagupte. My concern, as you've identified, is that we end up accidentally merging PRs that have been code reviewed because they don't have the "Waiting on factcheck" label.